src/test/java/uk/gov/hmcts/ccd/domain/service/casedataaccesscontrol/DefaultCaseDataAccessControlTest.java

```diff
@@ -224,6 +257,200 @@ void shouldGenerateAccessProfilesByCaseDetails() {
         verify(accessProfileService).generateAccessProfiles(anyList(), anyList());
     }
 
+    @Test
+    void shouldGenerateAccessProfilesByCaseAccessCategory_MatchCommaSeparatedLine() throws JsonProcessingException {
+        doReturn(USER_ID).when(securityUtils).getUserId();
+        CaseDetails caseDetails = new CaseDetails();
+        caseDetails.setCaseTypeId(CASE_TYPE_1);
+        caseDetails.setData(generateCaseData("UNSPEC_CLAIM"));
```

Comment:
Could we also add at least one test that confirms the `startsWith()` please, to ensure we have it the right way round :)

Reply:
done